src/config.ts

```diff
@@ -14,8 +14,7 @@ export const createConfigFile = async ({
   const config = {
     appId,
     appName,
-    bundledWebRuntime: true,
-    webDir: 'www',
+    webDir: 'dist',
```

Comment:
```suggestion
    bundledWebRuntime: false,
    webDir: 'dist',
```
In regular apps where capacitor is added we set bundledWebRuntime to false by default, so better set it to false than deleting the entry